portfolio/src/main/webapp/css/style.css

```diff
@@ -0,0 +1,58 @@
+@font-face {
+    font-family: selima;
+    src: url("../fonts/Selima\ .otf");
+}
+
+/* Header Section with navbar and image */
+#banner {
+    height: 100vh;
+    min-height: 500px;
```

Comment:
nice!

Reply:
Actually - was just thinking about this. The min-height of 500px might look funky on mobile browsers - but we can worry about that later :)